### ISSUE TITLE
Favicon Checker doesn't support non-ASCII characters

### ISSUE THREAD

**Issue by salvis** [RealFaviconGenerator/realfavicongenerator#265]
I have a site (url by email on request) that has a German umlaut in its site &lt;title&gt; attribute. The page is delivered as UTF-8, with proper
&lt;meta http-equiv="Content-Type" content="text/html; charset=utf-8" /&gt;
and the umlaut is encoded correctly as a two-character UTF-8 code point.

The site title is picked up for the Windows 8/10 preview, but Favicon Checker renders the 'ä' as 'XX'.

BTW, Favicon Generator and Favicon Checker are very helpful and work well with the Responsive Favicons Drupal module. Thank you!

**Comment by phbernard** [comment]
That's right. What you observe here is a dirty workaround to handle non-ASCII characters. But the checker should obviously support them.
